.gitignore

```diff
@@ -6,10 +6,39 @@ docs_html
 __*__
 
 dscaper_data/
-
-#audio files
 *.wav
 *.mp3
+*.png
```

Comment:
We do have pngs to be tracked, like images in the docs and in the README, a wouldn't ignore them. If you have pngs somewhere, may be better to ignore them inside that specific folder, but not globally.